src/components/new-request/search.vue

```diff
@@ -396,6 +459,9 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
   }
 
   get showEntityType (): boolean {
+    if (this.isConversion) {
+      return this.business != null && this.isPassValidation
```

Comment:
1. This should be `!==` (value and type comparison, even though null has no type because lint may complain)
2. This can actually be `return this.business && ...`

Also on line 504 below.